### PR TITLE
Foundation Email docset

### PR DESCRIPTION
**Pull request by joeworkman**
New docset

### CODE REVIEW

File: docsets/FoundationEmail/docset.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"author":{"name":"Joe Workman","link":"https://twitter.com/joeworkman"},"aliases":["Foundation for Email","Foundation","Email","Inky"],"archive":"FoundationEmail.tgz","version":"1.0.0","name":"Foundation for Email","specific_versions":[{"archive":"versions/1.0.0/FoundationEmail.tgz","version":"1.0.0"}]}
```
Comment (Kapeli) [inline docsets/FoundationEmail/docset.json]: Please have this in human-readable format (not minified).

Comment (Kapeli) [inline docsets/FoundationEmail/docset.json]: The versions should be the versions of the docs, not the docset itself. Foundation for Emails is at v2 as far as I can tell.

Also please rename to "Foundation for Emails".

**Comment by joeworkman** [comment]
I am pretty sure that I made all of your requested changes.

**Comment by Kapeli** [comment]
Thank you! 👍 

Sorry for the wait. I've got a bit of a backlog to get through 😞

**Pull request merged**